### ISSUE TITLE
Test failure with PG15: pointcloud-1.2.so: undefined symbol: pg_atoi

### ISSUE THREAD

**Issue by df7cb** [pgpointcloud/pointcloud#324]
```
13:38:33 --- /<<PKGBUILDDIR>>/pgsql/expected/pointcloud.out	2022-09-12 07:14:10.000000000 +0000
13:38:33 +++ /<<PKGBUILDDIR>>/pgsql/results/pointcloud.out	2022-09-12 11:38:33.031547414 +0000
13:38:33 @@ -1,12 +1,12 @@
13:38:33  set client_min_messages to ERROR;
13:38:33  SET extra_float_digits = 0;
13:38:33  CREATE EXTENSION pointcloud;
13:38:33 +ERROR:  could not load library "/<<PKGBUILDDIR>>/debian/postgresql-15-pointcloud/usr/lib/postgresql/15/lib/pointcloud-1.2.so": /<<PKGBUILDDIR>>/debian/postgresql-15-pointcloud/usr/lib/postgresql/15/lib/pointcloud-1.2.so: undefined symbol: pg_atoi
```

pg_atoi has been removed in February:

```
commit 73508475d69e90f98ebd9b7e1a5933a26a49c5e9
Author: Peter Eisentraut <peter@eisentraut.org>
Date:   Mon Feb 14 21:29:45 2022 +0100

    Remove pg_atoi()
    
    The last caller was int2vectorin(), and having such a general function
    for one user didn't seem useful, so just put the required parts inline
    and remove the function.
    
    Reviewed-by: John Naylor <john.naylor@enterprisedb.com>
    Discussion: https://www.postgresql.org/message-id/flat/b239564c-cad0-b23e-c57e-166d883cb97d@enterprisedb.com
```

**Comment by pblottiere** [comment]
Hello @df7cb,

Did you try with the master branch? Because I merged this PR last week: https://github.com/pgpointcloud/pointcloud/pull/318

**Comment by df7cb** [comment]
Hi Paul, I tried with 1.2.3. Can you tag a new release if master fixes it?

**Comment by pblottiere** [comment]
Hello Christoph,

> Can you tag a new release if master fixes it?

I just released a v1.2.4 with PostgreSQL 15 support.

BTW, do you plan to package Pointcloud >= v1.2.3 on Debian stable?

**Comment by df7cb** [comment]
Hi Paul,
everything is green now, thanks!

1.2.4 will be on stable in the sense that you can get it from apt.postgresql.org. The package in Bullseye itself will not be changed.
